### PR TITLE
docs(connectToggleRefinement): document render state

### PR DESCRIPTION
**Pull request by Haroenv**
**Summary**

adds docs to every key  of the renderState of toggle 

**Result**

Docs on all keys of the toggle refinement render state

This is exposed by TypeScript, and based on the docs I wrote in https://github.com/algolia/doc/pull/6551

### CODE REVIEW

File: src/connectors/toggle-refinement/connectToggleRefinement.ts [hunk 1]
```diff
@@ -77,14 +77,20 @@ const createSendEvent = ({
 };
 
 export type ToggleRefinementValue = {
-  /** whether this option is enabled */
+  /**
+   * whether this option is enabled.
```
Comment (Haroenv) [inline src/connectors/toggle-refinement/connectToggleRefinement.ts:81]:
```suggestion
   * Whether this option is enabled.
```

File: src/connectors/toggle-refinement/connectToggleRefinement.ts [hunk 2]
```diff
@@ -77,14 +77,20 @@ const createSendEvent = ({
...
+  /**
+   * Number of result if this option is toggled.
+   */
   count: number | null;
 };
 
 export type ToggleRefinementConnectorParams = {
-  /** Name of the attribute for faceting (eg. "free_shipping"). */
+  /**
+   * Name of the attribute for faceting (eg. "free_shipping").
```
Comment (sarahdayan) [inline src/connectors/toggle-refinement/connectToggleRefinement.ts:92]:
```suggestion
   * Name of the attribute for faceting (e.g., "free_shipping").
```

File: src/connectors/toggle-refinement/connectToggleRefinement.ts [hunk 3]
```diff
@@ -100,19 +106,42 @@ export type ToggleRefinementConnectorParams = {
...
     offFacetValue: ToggleRefinementValue;
   };
-  /** Creates an URL for the next state. */
+  /**
+   * Creates an URL for the next state.
+   */
   createURL: CreateURL<string>;
-  /** send a "facet clicked" insights event */
+  /**
+   * Send a "facet clicked" insights event.
```
Comment (sarahdayan) [inline src/connectors/toggle-refinement/connectToggleRefinement.ts:135]:
```suggestion
   * Send a "Facet Clicked" Insights event.
```

**Review by sarahdayan — APPROVED** [review]
Non-blocking nitpicks.

**Review by francoischalifour — APPROVED** [review]

**Pull request merged**